docs/development/releasing.md

```diff
@@ -5,19 +5,50 @@ This document describes the process for releasing a new version of Electron.
 ## Determine which branch to release from
 
 - **If releasing beta,** create a new branch from `master`.
-- **If releasing a stable version,** create a new branch from the beta branch you're stabilizing.
+- **If releasing a stable version,** create a new branch from the beta branch
+  you're stabilizing.
 
 ## Find out what version change is needed
 Run `npm run prepare-release -- --notesOnly` to view auto generated release
-notes.  The notes generated should help you determine if this is a major, minor,
-patch, or beta version change. Read the
-[Version Change Rules](../tutorial/electron-versioning.md#version-change-rules) for more information.
+notes. The notes generated should help you determine if this is a major,
+minor, patch, or beta version change. Read the
+[Version Change Rules](../tutorial/electron-versioning.md#version-change-rules)
+for more information.
+
+**NB:** If releasing from a branch, e.g. 1-8-x, check out the branch with
+`git checkout 1-8-x` rather than `git checkout -b remotes/origin/1-8-x`.
+The scripts need `git rev-parse --abbrev-ref HEAD` to return a short name,
+e.g. no `remotes/origin/`
+
+## Set your tokens and environment variables
+The Electron S3 Bucket in LastPass has environment variables needed for the
```

Comment:
@ckerr given that this is a public facing document, I don't think we should explicitly document GitHub process here.  Maybe a generic statement about you'll need Electron S3 credentials?